plugins/xpressive/.gitignore

```diff
@@ -0,0 +1,2 @@
+#ignore the patched file
+exprtk.patched.hpp
```

Comment:
The patch was removed due to compilation with the -fexceptions flag and the release of new exprtk version that allow one to disable return statement.